.github/workflows/tests.yml

```diff
@@ -38,7 +38,7 @@ jobs:
 
       - name: Run tests with coverage
         run: |
-          yarn coverage --ci --reporters github-actions --max-workers ${{ steps.cpu-cores.outputs.count }} ./spec/${{ matrix.specs }}
+          yarn coverage --ci --reporters github-actions '--reporters=<rootDir>/spec/slowReporter.js' --max-workers ${{ steps.cpu-cores.outputs.count }} ./spec/${{ matrix.specs }}
```

Comment:
IMO this reporter shouldn't be used in PRs unless it was to fail PRs based on runtime. Though on develop runs it'd make sense.

Reply:
@t3chguy I made it run only on develop branch